apps/admin-x-settings/src/components/settings/general/UserDetailModal.tsx

```diff
@@ -417,6 +420,7 @@ const UserDetailModalContent: React.FC<{user: User}> = ({user}) => {
             size={canAccessSettings(currentUser) ? 'md' : 'bleed'}
             stickyFooter={true}
             testId='user-detail-modal'
+            topRightContent={topRightContent}
```

Comment:
This could be simplified by directly injecting `{<></>}` to `topRightContent.

Reply:
@peterzimon This update has been made in [813d0d4](https://github.com/TryGhost/Ghost/pull/24510/commits/813d0d45d0ec896a462eedd13d5ff0bc5a788500).